src/tests/Equinor.ProCoSys.IPO.Domain.Tests/AggregateModels/InvitationAggregate/InvitationTests.cs

```diff
@@ -279,13 +318,136 @@ public void RemoveParticipant_ShouldThrowException_WhenParticipantNotGiven()
         public void RemoveAttachment_ShouldThrowException_WhenAttachmentNotGiven()
             => Assert.ThrowsException<ArgumentNullException>(() => _dutWithMcPkgScope.RemoveAttachment(null));
 
+        [TestMethod]
+        public void RemoveAttachment_ShouldRemoveAttachment()
+        {
+            Assert.AreEqual(1, _dutWithMcPkgScope.Attachments.Count);
+            Assert.AreEqual(_attachment, _dutWithMcPkgScope.Attachments.First());
+
+            _dutWithMcPkgScope.RemoveAttachment(_attachment);
+
+            Assert.AreEqual(0, _dutWithMcPkgScope.Attachments.Count);
+            Assert.IsInstanceOfType(_dutWithMcPkgScope.DomainEvents.Last(), typeof(AttachmentRemovedEvent));
```

Comment:
I think the assert of correct type of event deserve its own test: RemoveAttachment_ShouldAddRemoveAttachmentEvent()
This goes for all asserts of same type in this testclass